src/xsar/sentinel1_meta.py

```diff
@@ -663,6 +663,12 @@ def image(self):
         img_dict['history'] = self.xml_parser.get_compound_var(self.files['annotation'].iloc[0], 'image', describe=True)
         return img_dict
 
+    @property
+    def azimuth_fmrate(self):
```

Comment:
This property has to be documented in the user manual, so it need a docstring

Reply:
doc string added.